Gemfile

```diff
@@ -29,4 +29,5 @@ end
 gem "wdm", "~> 0.1.1", :platforms => [:mingw, :x64_mingw, :mswin]
 
 gem "nokogumbo"
+gem "webrick"
```

Comment:
@Matt-Blanco: I'm trying to figure out what this gem is and how it relates to this PR, could you help me out? Afaict, it starts an HTTP Server?

Reply:
I'm not too sure where the discrepancy between gemfiles arose from I just did `gem install` once I installed Ruby and began working on the addition to the paper. The changes to the Gemfile and Gemfile.lock can probably be omitted

Reply:
I've needed to add the webrick gem as a bug fix for Ruby 3.0. See [here](https://github.com/jekyll/jekyll/issues/8523).

Reply:
Ah good to know, thanks @alundgard!